x-pack/plugins/uptime/server/lib/requests/get_journey_screenshot.ts

```diff
@@ -59,25 +58,42 @@ export const getJourneyScreenshot: UMElasticsearchQueryFn<
           image: {
             top_hits: {
               size: 1,
-              _source: ['synthetics.blob', 'synthetics.blob_mime', 'synthetics.step.name'],
+              _source: [
+                '@timestamp',
+                'monitor.check_group',
+                'screenshot_ref',
+                'synthetics.package_version',
+                'synthetics.step',
+                'synthetics.type',
+                'synthetics.blob',
+                'synthetics.blob_mime',
+                'synthetics.step.name',
+              ],
             },
           },
         },
       },
     },
   };
-  const { body: result } = await uptimeEsClient.search({ body: params });
 
-  if (result?.hits?.total.value < 1) {
-    return null;
+  const result = await uptimeEsClient.search({ body });
+
+  const decoded = ResultWrapperType.decode(result.body.aggregations?.step.image.hits.hits ?? null);
+
+  if (!isRight(decoded)) throw Error('Error parsing journey screenshot type. Malformed data.');
```

Comment:
again question here about why we need this validation.

Reply:
Fixed: https://github.com/elastic/kibana/pull/101678/commits/f2e7880875c03dacef3032987edb882270038d4d.